cargo-dist/tests/gallery/dist/snapshot.rs

```diff
@@ -169,7 +169,7 @@ pub fn snapshot_settings_with_gallery_filter() -> insta::Settings {
         r#""build_environment": "indeterminate""#,
     );
     settings.add_filter(
-        r"[0-9a-f]{64}  (?<filename>([a-zA-Z0-9-_]+)(\.tar\.gz|\.pkg))",
+        r"[0-9a-f]{64} .(?<filename>([a-zA-Z0-9-_]+)(\.tar\.gz|\.pkg))",
```

Comment:
needed because we changed to actually emitting ` *` (space star)  instead of `  ` (space space)